sound/soc/sdca/sdca_functions.c

```diff
@@ -1222,6 +1222,92 @@ static int find_sdca_entity_ge(struct device *dev,
 	return -EINVAL;
 }
 
+static int find_sdca_entity_hide(struct device *dev, struct fwnode_handle *function_node,
+	struct fwnode_handle *entity_node, struct sdca_entity *entity)
+{
+	struct sdca_entity_hide *hide = &entity->hide;
+	unsigned int delay, num_af, af_list[SDCA_MAX_FUNCTION_COUNT];
+	int nval, ret;
+	int num_hid_desc, num_report_decs;
+	struct hid_descriptor hid_desc;
+	unsigned char *report_desc = NULL;
```

Comment:
nitpick: can you resort these and combine the same types

Reply:
will fix

Reply:
and one line per variable is you have a mix of array, pointers and scalars.